kinde-j2ee-app/target/kinde-j2ee-app/WEB-INF/web.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is the target folder necessary to be merged? Can it not be deleted. It is created during a build.

Reply:
Removed